### ISSUE TITLE
Bad behavior when lacking resource files

### ISSUE THREAD

**Issue by felipesanches** [agrande/lba2remake#145]
I did run the dev-server without the original game files (on purpose to see what happens).

![screenshot at 2018-03-03 09 28 02](https://user-images.githubusercontent.com/213676/36934224-3c6599ce-1ec5-11e8-9338-d17d9e926c3f.png)

The message `Assets compiled successfully in 45230 ms :-)` is misleading because there are no assets available in my setup. Also, it did "something" during 45 painful seconds during which I did not now it was trying to read the resource files and the web-browser was unable to show me any info (localhost:8080 did not load at all).

I'd suggest adding a message such as: "Looking for game files to unpack, please wait." immediately before starting to do so, so that the user is aware of what is going on.

**Comment by felipesanches** [comment]
Also this was surprising to me:

![screenshot at 2018-03-03 09 33 14](https://user-images.githubusercontent.com/213676/36934304-f173bca6-1ec5-11e8-8304-2e79ea9d511c.png)


How come do I see a Twinsen image if I do not have any assets in my www/data directory? This looks like a potential copyright infringement. Can't this image be loaded from the game assets instead of hosted in the project git repo?

**Comment by felipesanches** [comment]
Also, the message "Error: no such file or directory" does not specify which file it was looking for, which would be very useful to know.

**Comment by agrande** [comment]
Ok, there's a lot of information in this issue, let me reply one point at a time.
The message `Assets compiled successfully in 45230 ms :-)` has nothing to do with game assets, it's originating from the webpack-express-middleware dependency.

Basically webpack is taking all our source files and transpiles them to a javascript bundle (this is something I'm planning to document soon, see #7).

So here asset means source file. And the 45 seconds is the time it takes to compile the sources on your PC. It's taking much less than that on a recent computer.

**Comment by agrande** [comment]
> I'd suggest adding a message such as: "Looking for game files to unpack, please wait." immediately before starting to do so, so that the user is aware of what is going on.

As indicated, this is actually not what's going on here.

**Comment by agrande** [comment]
Regarding the loader image, yes technically, this is a copyright infringement, and it would be a good idea to load it from the game assets. However since this image is widely available over the web and just a loader image, it sounds to me as benign infringement. Not sure it's worth the effort spending the time to fix it at this point. In the future, when/if more people notice this project it would make sense.

**Comment by agrande** [comment]
> Also, the message "Error: no such file or directory" does not specify which file it was looking for, which would be very useful to know.

Again, this is [webpack](https://www.npmjs.com/package/webpack-express-middleware) spitting this message when facing a 404. The file missing can easily be found from the devtools in the browser.

**Comment by agrande** [comment]
I opened a new issue for the loader image: #149. It's set to a further milestone, but if you feel like working on it now, please go ahead ;)

Regarding the other points, I think I'll just describe the webpack thing in the docs. No need to hack into it to change the messages it displays. That wouldn't be a good use of our time. Free time is a very limited resource!

**Comment by xesf** [comment]
The load imaged is an upscaled image using waifu2x.
Menu image has been enhaced as well and mirrored to have a wider image.
